src/events/EventsClient.ts

```diff
@@ -3,6 +3,12 @@ import { FronteggAuthenticator } from '../authenticator';
 import { config } from '../config';
 import Logger from '../helpers/logger';
 import { channelTypes, ITriggerOptions } from './types';
+import { IEventStatuses } from './types/EventStatuses.interface';
+
+const POLLING_LIMIT = 20;
+const POLLING_TIMEOUT = 3 * 1000;
```

Comment:
Let's increase the rate here.

First one should fire after 400ms.
Then increase in 20% values